include/triton/Dialect/Triton/IR/TritonOps.td

```diff
@@ -438,4 +467,45 @@ def TT_AssertOp : TT_Op<"assert", [MemoryEffects<[MemWrite]>]> {
   let assemblyFormat = "$condition `,` $message `,` $file `,` $func `,` $line attr-dict `:` type($condition)";
 }
 
+//
+// Make a Tile Pointer
+//
+def TT_MakeTilePtrOp : TT_Op<"make_tile_ptr",
```

Comment:
Let's call it `MakeTensorPtr`. `make_tile_ptr` is a good name for the `triton.language` python frontend because it disambiguates triton tensor and torch tensors, but at this level of the IR we only have triton tensors

Reply:
Fixed. Also renamed all frontend usages into "block" and "tensor" for backend (as said in the Slack channel). Now seems we have:
- "block" for the user frontend
- "tensor" for the backend
- "tile" for the hardware scheduling